### ISSUE TITLE
UI Going Blank

### ISSUE THREAD

**Issue by ManApart** [lVlyke/starfield-mod-loader#2]
I'm really excited to use this. I hadn't realized that Vortex wasn't supported on linux, and so to have a "native" tool is very exciting.

Unfortunately my window is going blank  while trying to enter my directories. Initially everything looks fine, but after 15ish seconds, the whole display goes blank:

![image](https://github.com/lVlyke/starfield-mod-loader/assets/4498895/493eaa7d-d6a8-4b10-9285-cb96d641cf95)

Running in the terminal, I see this when trying to go to preferences while the screen is blank:
```
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94495)
    at WebContents.send (node:electron/js2c/browser_init:2:79721)
    at click (/media/austin/extradrive1/games/starfield-additional/programs/starfield-mod-loader-3-1/resources/app/electron.js:416:66)
    at MenuItem.click (node:electron/js2c/browser_init:2:30958)
    at Menu._executeCommand (node:electron/js2c/browser_init:2:36354)
```

I'm running the latest version of pop os.

**Comment by lVlyke** [comment]
Hello, thanks for the bug report.

I was able to reproduce this issue and resolved it by downgrading Electron to an earlier version, as the latest version appears to have an issue that causes the renderer to crash on Linux.

I just released [version 0.3.2](https://github.com/lVlyke/starfield-mod-loader/releases/tag/v0.3.2) which should resolve this issue.

**Comment by ManApart** [comment]
This resolved the screen going blank. Thank you!
